Xamarin.Essentials/Share/Share.shared.cs

```diff
@@ -86,16 +108,42 @@ public ShareFileRequest(string title, FileBase file)
         }
 
         public ShareFileRequest(ShareFile file)
+            => File = file;
+
+        public ShareFileRequest(FileBase file)
+            => File = new ShareFile(file);
+
+        public ShareFile File { get; set; }
+    }
+
+    public class ShareMultipleFilesRequest : ShareRequestBase
+    {
+        public ShareMultipleFilesRequest()
         {
-            File = file;
         }
 
-        public ShareFileRequest(FileBase file)
+        public ShareMultipleFilesRequest(IEnumerable<ShareFile> files, string title = null)
         {
-            File = new ShareFile(file);
+            Files = files;
+            Title = title;
         }
 
-        public ShareFile File { get; set; }
+        public ShareMultipleFilesRequest(IEnumerable<FileBase> files, string title = null)
```

Comment:
Can we swap this to put the title first to match the single file? I know we are not consistent with the text, but I can still cry at night.
I know this is nice for default parameters, but I think consistency might be better. We might just need another overload without. Not perfect, but it is what we have now.

Reply:
I'll do it 😭

Reply:
😭